apis/ec2/v1beta1/referencers.go

```diff
@@ -209,19 +207,19 @@ func (mg *NATGateway) ResolveReferences(ctx context.Context, c client.Reader) er
 	mg.Spec.ForProvider.SubnetID = aws.String(subnetIDResponse.ResolvedValue)
 	mg.Spec.ForProvider.SubnetIDRef = subnetIDResponse.ResolvedReference
 
-	// // Resolve spec.elasticIp
-	AllocationIDRespone, err := r.Resolve(ctx, reference.ResolutionRequest{
+	/*// // Resolve spec.elasticIp
```

Comment:
Is this intentionally commented out?

Reply:
sigh. had to comment it out to get generate going. am fixing